vtrace/tests/__init__.py

```diff
@@ -27,7 +27,7 @@ def tearDown(self):
             self.proc.stdin.flush()
             self.trace.run()
         try:
-            self.proc.wait(timeout=30)
+            self.proc.wait(timeout=120)
```

Comment:
Heads up, git might complain here depending on if you merge this PR or the other one first.

Reply:
surprisingly, it allowed me to merge in updates without complaining so far....  it'd be way cool is the unittests passed *and* it still allowed me to merge.  will let you know.  <git-merge-roulette>